Detectors/ITSMFT/MFT/base/include/MFTBase/Geometry.h

```diff
@@ -76,7 +76,9 @@ class Geometry : public TNamed
   static const Double_t sGlueEdge;
   static const Double_t sShiftDDGNDline;
   static const Double_t sShiftline;
-  static const Double_t sRohacell;
+  static const Double_t mGlueRohacellCarbonThickness;
+  static const Double_t mKaptonOnCarbonThickness;
+  static const Double_t mKaptonGlueThickness;
```

Comment:
please use `s` and not `m` for these variables.

Reply:
Hello, it's done now.